cloud_run_service_deploy_tag/test.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: BlueprintTest
+metadata:
+  name: cloud_run_service_deploy_tag
+spec:
+  skip: true
```

Comment:
I noticed you skipped a few ~15ish tests. What the reason for doing that?

Reply:
These were generated from the MM configs, for example
https://github.com/GoogleCloudPlatform/magic-modules/blob/e0dec232ee04c54a8956101917b13a66d98544af/mmv1/products/cloudrun/terraform.yaml#L248 was skipped there.